templates/my-package.hbs.js

```diff
@@ -0,0 +1,3 @@
+export default function someFunction() {
```

Comment:
Curious to hear what others have to say. Should we make the autogenerated package a typescript project like [quilts](https://github.com/Shopify/quilt/tree/master/templates)? I know most of the packages in this repo are .js packages but moving forward we may want to encourage typescript projects.

Reply:
Yes, my plan is to migrate every package into typescript. But for now, I didn't want to migrate everything in one PR. I will do this in a follow-up 😄